Formula/borg.rb

```diff
@@ -0,0 +1,31 @@
+# Documentation: https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class Borg < Formula
+  desc "A terminal based search engine for bash commands"
+  homepage "https://github.com/crufter/borg"
+  url "https://github.com/crufter/borg/releases/download/v0.0.1/borg_darwin_amd64"
+  version "0.0.1"
+  sha256 "69e75c846fa4212eaf6be26b5ff49266afcd9a46712948a16b1d406da51d8ddf"
+
+  def install
+    # ENV.deparallelize  # if your formula fails when building in parallel
+
+    system "mv", "#{bin}/borg_darwin_amd64", "#{bin}/borg"
```

Comment:
This will disappear since we need to build from source but there’s nothing in `bin` (it doesn’t even exist) when the install step starts. You probably meant `"bin/borg_darwin_amd64"` instead of `"#{bin}/borg_darwin_amd64"`. In this case the correct command would have been:

```rb
bin.install "bin/borg_darwin_amd64" => "borg"
```